.pre-commit-config.yaml

```diff
@@ -94,7 +94,3 @@ repos:
         args:
           - -w
           - -s
-  - repo: https://github.com/rhysd/actionlint
-    rev: v1.7.0
-    hooks:
-      - id: actionlint
```

Comment:
Why are we removing the actionlint precommit?

Reply:
Because it exists already, 12 lines up